cli/lib/kontena/cli/stacks/registry/show_command.rb

```diff
@@ -0,0 +1,28 @@
+require_relative '../common'
+
+module Kontena::Cli::Stacks::Registry
+  class ShowCommand < Kontena::Command
+    include Kontena::Cli::Common
+    include Kontena::Cli::Stacks::Common
+    include Kontena::Cli::Stacks::Common::StackNameParam
+
+    banner "Shows information about a stack on the stacks registry"
+
+    option ['-v', '--versions'], :flag, "Only list available versions"
```

Comment:
Should we leave this now as is, and really implement it in another PR? I'd really like to get this merged :)